src/mozanalysis/frequentist_stats/linear_models/functions.py

```diff
@@ -17,6 +18,35 @@
 
 logger = logging.getLogger(__name__)
 
+##Types
+
+ComparativeOption = Literal["individual", "comparative"]
```

Comment:
Typically I'd choose to use an Enum here (something like below), is Literal a better option in this case?

```py
class ComparativeOption(str, Enum):
    INDIVIDUAL = "individual"
    COMPARATIVE = "comparative"
````

Reply:
tbh, I feel like an Enum is probably better. That way Jetstream can even import and use it.

Reply:
Done